src/app/containers/LoginContainer.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { loginAction } from '../../redux/actions/LoginActions';
+import Login from '../components/Login';
+
+export class LoginContainer extends Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errors: {}
+    };
+  }
+  onChange = (event) => {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value });
+  }
+  checkValidEmail = (userEmail) => {
```

Comment:
I think we can put such methods that check validation (and aren't reliant on a class) in a utils folder because they will be used by signup too. Discuss with @sanya-kenneth.